.github/workflows/scan-for-cves.yaml

```
@@ -41,3 +41,12 @@
         with:
           sarif_file: reports
           # TODO: github dependency tree?
+
+  createIssue:
+    runs-on: ubuntu-latest
+    name: Create Issue
+    if: steps.generateSarifReports.conclusion == 'failure'
```

actionlint (1.7.7)

context "steps" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)